src/Xamarin.Android.Build.Tasks/Utilities/Files.cs

```diff
@@ -329,6 +329,21 @@ public static bool IsPortablePdb (string filename)
 				return false;
 			}
 		}
+
+		internal static string QuoteFileName (string fileName)
+		{
+			var builder = new CommandLineBuilder ();
+			builder.AppendFileNameIfNotNull (fileName);
+			return builder.ToString ();
+		}
+
+		internal static string GetShortPath (string path)
+		{
+			if (Environment.OSVersion.Platform != PlatformID.Win32NT)
+				return QuoteFileName (path);
+			var shortPath = KernelEx.GetShortPathName (Path.GetDirectoryName (path));
```

Comment:
Why only get the short path name for the *directory*, and not the *entire path*?

Reply:
Can't remember.. its what we did